Makefile

```diff
@@ -37,6 +37,7 @@ checks:
 	./scripts/checks "$(PKGS)"
 
 run-build: deps checks clean
+	cd ${GOPATH}/src/github.com/contiv/netplugin && version/generate_version  && \
```

Comment:
this is minor, feel free to ignore.

Not sure if you already explored this, but this looks like a good candidate for `go generate` (https://blog.golang.org/generate)

so we can add something like `//go:generate <script similar to version/generate_version>` in say an existing the `netd.go` and `netmaster.go` (and even docker and k8s plugins) files to generate their respective version logic. It's kind of nice in a way that this will happen part of compilation without extra Makefile instrumentation and keep version logic close to each daemon we produce as part of the build.